includes/wikia/Extensions.php

```diff
@@ -94,6 +94,13 @@
 	F::app()->registerApiController( 'WikisApiController', "{$IP}/includes/wikia/api/WikisApiController.class.php" );
 }
 
+// During migration drop parser expiry on non-English wikis to 24 hours (PLATFORM-3765)
+if ( !wfHttpsAllowedForURL( $wgServer ) ||
+	( wfHttpsEnabledForURL( $wgServer ) && $wgLanguageCode !== 'en' )
```

Comment:
What's the reason to keep it that short after migration? Just in case we find something else?